src/ui/preview/Preview.vue

```diff
@@ -40,12 +40,6 @@
                     :current-view="currentView"
                     @setView="setView"
                 />
```

Comment:
Thanks for removing this, I am also working on it and tracked in https://github.com/nasa/openmct/issues/2821